docs/CONTRIBUTING.md

```diff
@@ -249,3 +250,48 @@ These contribution guidelines were inspired by and are based on Atom’s
 contribution guidelines. They were modified for the purposes of this repository.
 https://github.com/atom/atom/blob/master/CONTRIBUTING.md - Copyright (c)
 2011-2017 GitHub Inc. (MIT)
+
+### JSDoc Styleguide
+
+For code documentation, we use [JSDoc](http://usejsdoc.org/).
+You can generate a HTML documentation out of it.
+Navigate to the projects folder and run:
```

Comment:
This seems unnecessary. But we need something like `To build the documentation site, run the following command:`.